runtimes/api.py

```
@@ -98,9 +106,34 @@ async def refresh_runtime_health() -> dict:
 
 
+_RICH_POLICY_KEY = "ui_routing_policy"
+
+
+def _load_rich_policy() -> dict:
+    """Return the persisted rich UI policy (pools + policy + triggers), or {}."""
+    try:
+        from webui.config_store import JsonConfigStore
+        store = JsonConfigStore()
+        return store.load(_RICH_POLICY_KEY) or {}
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)